src/components/Navigation/ToggleButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+
+import styles from './ToggleButton.scss';
+
+const ToggleButton = React.createClass({
+
+  getInitialState() {
+    return {
+      from: this.props.from,
```

Comment:
Det er en react "anti-pattern" å initialisere state fra props:
https://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html